iModelCore/libsrc/rapidjson/vendor/include/rapidjson/writer.h

```diff
@@ -547,7 +547,9 @@ inline bool Writer<StringBuffer>::WriteDouble(double d) {
     if (internal::Double(d).IsNanOrInf()) {
         // Note: This code path can only be reached if (RAPIDJSON_WRITE_DEFAULT_FLAGS & kWriteNanAndInfFlag).
         if (!(kWriteDefaultFlags & kWriteNanAndInfFlag))
```

Comment:
this code is in this file in 2 places. You should make this change in both of them.